index.html

```diff
@@ -134,12 +134,16 @@ <h2></h2>
           <p class="keywords"></p>
         </article>
       </template>
-      <nav>
-        <button id="previous-page-button"><</button>
-        <button id="next-page-button">></button>
-      </nav>
     </div>
   </main>
+  <div id="page-bottom">
+    <article id="placeholder-article" class="article">
```

Comment:
by the way, are you doing BEM or any other class naming scheme?

Reply:
Not at the moment. I looked into BEM and some other schemes when I was first learning CSS, but trying to put them into practice created too much friction (and frustration) in the learning process, so I decided to drop them. I imagine BEM would make more sense to me if I was working in a larger code base where it was already being used, but trying to start using it from 0 on smaller projects just didn't click for me.

Reply:
no worries. I would still recommend to think about when you use which kind of class. in this particular block (with its contents) I can not infer the logic behind your choice.

Reply:
Okay, maybe we could set aside some time to talk about this in more depth? It's been a nagging pain point from the beginning. I'll send you an email.